core/src/com/rallyhealth/weepickle/v1/core/Types.scala

```diff
@@ -219,6 +219,20 @@ trait Types { types =>
         }
       }
     }
+
+    /*
+     * Used by Radix-based visitKeyValue. Excludes safety checks for speed's sake, since generated client ensures values are in range
+     * (otherOffset >= 0) && (csOffset >= 0) && (csOffset <= cs.length - compareLen) && (otherOffset <= other.length - compareLen) &&
+     */
+    protected def regionMatches(cs: CharSequence, csOffset: Int, other: CharSequence, otherOffset: Int, compareLen: Int): Boolean = {
```

Comment:
I could use us needing this in other places. Once it's in `CaseR`, I don't think MiMa will let us remove it.

Would it make sense to toss this into `Util.scala` instead?